storage/connect/zip.c

```diff
@@ -1066,12 +1066,6 @@ extern int ZEXPORT zipOpenNewFileInZip4_64 (zipFile file, const char* filename,
     uInt i;
     int err = ZIP_OK;
 
-#    ifdef NOCRYPT
-    (crcForCrypting);
-    if (password != NULL)
-        return ZIP_PARAMERROR;
-#    endif
-
```

Comment:
restore the `ifdef` block here, the warning was about the `(crcForCrypting);` statement (this should be removed).

Reply:
Done.